CMakeLists.txt

```diff
@@ -38,32 +38,26 @@ option(MINIMAL_BUILD "Produce a minimal build with only the essential features (
 option(MUSL_OPTIMIZED_BUILD "Enable if you want a musl optimized build" OFF)
 
 include(GNUInstallDirs)
+
 # Add path for custom CMake modules.
 list(APPEND CMAKE_MODULE_PATH
 	"${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules")
 
-include(GetGitRevisionDescription)
+include(versions)
 
+# Libs version
 if(NOT DEFINED FALCOSECURITY_LIBS_VERSION)
-	# Try to obtain the exact git tag
-	git_get_exact_tag(LIBS_TAG)
-	if(NOT LIBS_TAG)
-		# Obtain the closest tag
-		git_describe(FALCOSECURITY_LIBS_VERSION "--always" "--tags" "--abbrev=7")
-		# Fallback version
-		if(FALCOSECURITY_LIBS_VERSION MATCHES "NOTFOUND$")
-			set(FALCOSECURITY_LIBS_VERSION "0.1.1dev")
-		endif()
-		# Format FALCOSECURITY_LIBS_VERSION to be semver with prerelease and build part
-		string(REPLACE "-g" "+" FALCOSECURITY_LIBS_VERSION "${FALCOSECURITY_LIBS_VERSION}")
-	else()
-		# A tag has been found: use it as the Falco version
-		set(FALCOSECURITY_LIBS_VERSION "${LIBS_TAG}")
-	endif()
+	get_libs_version(FALCOSECURITY_LIBS_VERSION)
+endif()
+
+message(STATUS "Libs version: ${FALCOSECURITY_LIBS_VERSION}")
+
+# Driver version
+if(NOT DEFINED DRIVER_VERSION)
+	get_drivers_version(DRIVER_VERSION)
 endif()
```

Comment:
I'd rather call this FALCOSECURITY_DRIVER_VERSION really :)

Reply:
I know we have been calling it DRIVER_VERSION and i think changing it would break backward compatibility though.